src/web/Link.tsx

```diff
@@ -37,7 +37,14 @@ const _styles = {
     }
 };
 
+const _longPressTime = 1000;
+
 export class Link extends RX.Link<void> {
+    
+    private _lastMouseDownEvent: Types.SyntheticEvent;    
```

Comment:
Grrrr. Some left overs from another thing I wanted to include but not right now.